src/main/java/com/iota/iri/utils/thread/ThreadUtils.java

```diff
@@ -25,7 +26,7 @@
      * @see #spawnThread(Runnable, ThreadIdentifier)
      * @see #stopThread(ThreadIdentifier)
      */
-    private static final HashMap<Object, Thread> threads = new HashMap<>();
+    private static final Map<Object, Thread> threads = new HashMap<>();
```

Comment:
I thought we will get rid of this class in later PR.
Of course I don't object to the change.